app/policies/application_policy.rb

```diff
@@ -8,6 +8,7 @@ class ApplicationPolicy < ActionPolicy::Base
 
   def authenticated?
     deny! if doorkeeper_token.nil?
+    true
   end
```

Comment:
```suggestion
  def authenticated?
    deny! if doorkeeper_token.nil?
  end
```

Tu peux enlever le true du coup :p